### PR TITLE
pr_20

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 3 issues

File: src/main/java/org/sonarqube/A.java [hunk 1]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java]: ###AWAS2GSXg9QtoM543Hdt###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_20@6
Complete the task associated to this TODO comment.

File: src/main/java/org/sonarqube/A.java [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
+    message.length();
+
     return message;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAS2GSXg9QtoM543Hdu###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_20@9
The return value of "length" must be used.

Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAS2GSXg9QtoM543Hdv###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_20@9
A "NullPointerException" could be thrown; "message" is nullable here.

File: src/main/java/org/sonarqube/A.java (hunk 1 above)
Comment (sns-seb) [inline src/main/java/org/sonarqube/A.java]: simulating user comment on about to be deleted comment

Comment (sns-seb2) [reply src/main/java/org/sonarqube/A.java]: simulating bot comment not autogenerated

Comment (sns-seb) [reply src/main/java/org/sonarqube/A.java]: let's do another user comment, just because

**Review by sns-seb2 — CHANGES_REQUESTED** [review]
SonarQube has detected 2 issues

**Review by sns-seb2 — COMMENTED** [review]
Test

**Review by sns-seb2 — COMMENTED** [review]
Test with 1 comment

File: src/main/java/org/sonarqube/A.java [hunk 3]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // fix this
   public String foo() {
+    message = null;
+    message.length();
+
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:10]: comment test 1

**Pull request closed without merging**